cmd/expiration-mailer/main.go

```diff
@@ -168,13 +174,13 @@ func (m *mailer) findExpiringCertificates() error {
 			 AND cert.expires > :cutoffA
 			 AND cert.expires <= :cutoffB
 			 AND cs.status != "revoked"
-			 AND cs.lastExpirationNagSent <= :nagCutoff
+			 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
```

Comment:
`lastExpirationNagSent` is set to NOT NULL in the schema and when a new certificateStatus is added the value is set to `0000-00-00 00:00:00` -- so we should be able to skip the `COALESCE`.